server/routes/ticket.py

```
@@ -141,8 +139,8 @@ def get(self):
                 status_code=200,
             )
 
-        except Exception as e:
-            current_app.logger.error(f"Error fetching tickets: {str(e)}")
+        except Exception:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)